brage-import/src/main/java/no/sikt/nva/brage/migration/merger/PreMergeValidator.java

```diff
@@ -1,37 +1,27 @@
 package no.sikt.nva.brage.migration.merger;
 
-import java.util.List;
-import no.unit.nva.model.AdditionalIdentifier;
 import no.unit.nva.model.Publication;
 import no.unit.nva.model.contexttypes.Degree;
 import nva.commons.core.JacocoGenerated;
 
 public final class PreMergeValidator {
 
-    public static final String HANDLE_ALREADY_EXIST_ERROR_MESSAGE = "Cannot merge student degree with nva publication"
-                                                                    + " with handle in additional identifiers: %s";
-
     @JacocoGenerated
     private PreMergeValidator() {
 
     }
 
-    public static void validate(Publication existingPublication) {
-        var handles = handleInAdditionalIdentifier(existingPublication);
-        if (isDegree(existingPublication) && !handles.isEmpty()) {
-            throw new DegreeMergingException(String.format(HANDLE_ALREADY_EXIST_ERROR_MESSAGE, handles));
-        }
+    public static boolean shouldNotMergePublications(Publication existingPublication) {
```

Comment:
This method should now be renamed to something else.

Reply:
What about shouldNotMergeMatadata? So we differentiate between "merging metadata" vs "injecting handle only"